src/template.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone project</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <h2>Mnet</h2>
+        </div>
+        <ul>
+          <li>
+            <a href="">movies(<span class="total"></span>)</a>
+          </li>
+          <li>
+            <a href="">new movies</a>
+          </li>
+          <li>
+            <a href="">old movies</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
- [ ] Can you please work more a bit on the typography of the header contents? The contrast between the text color and the background-color is too close, and I think the font-size supposed to be increased as well. 🙏